Dockerfile.special-resource-operator-build

```diff
@@ -10,7 +10,7 @@ RUN yum install -y which
 RUN yum install -y podman docker
 RUN curl -L "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" -o /tmp/kubectl
 RUN install -o root -g root -m 0755 /tmp/kubectl /usr/local/bin/kubectl
-RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.42.1
+RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.1
```

Comment:
In https://github.com/openshift-psap/special-resource-operator/pull/177 I am using `1.43.0` can we make this the version in skipper too?

Reply:
fixed